sips/sip-010/sip-010-fungible-token-standard.md

```diff
@@ -0,0 +1,172 @@
+# Preamble
+
+Sip Number: 010
+
+Title: Standard Trait Definition for Non-Fungibe Tokens
+
+Author: Hank Stoever (hstove@gmail.com)
+
+Consideration: Technical
+
+Type: Standard
+
+Status: Draft
+
+Created: 25 January 2021
+
+License: CC0-1.0
+
+Sign-off:
+
+Layer: Traits
+
+Discussions-To: https://github.com/stacksgov/sips
+
+# Abstract
+
+Fungible tokens are digital assets that can be sent, received, combined, and divided. Most forms of currency and cryptocurrencies are fungible tokens. They have become a building block of almost all blockchains. This SIP aims to provide a flexible and easy-to-implement standard that can be used by developers on the Stacks blockchain when creating their own tokens.
+
+# License and Copyright
+
+This SIP is made available under the terms of the Creative Commons CC0 1.0 Universal license, available at https://creativecommons.org/publicdomain/zero/1.0/
+This SIP’s copyright is held by the Stacks Open Internet Foundation.
+
+# Introduction
+
+Digital assets can be divided into two categories, based on the token's fungibility. A _fungible_ token can be broken down into small units and added together. An owner of a fungible asset only needs to care about their balance, that is, the total amount of a particular fungible asset that they own. Most well-known currencies are fungible.
+
+For example, if a user owns 10 units of a fungible asset, they may send 2 units to a different user. At this point, their balance is 8 units. If they later receive more units, their total balance will be updated. For fungible tokens, there is no difference between any two different amounts of the fungible token.
```

Comment:
"For fungible tokens, there is no difference between any two different amounts of the fungible token." This should go somewhere before the example as it is the main definition, I think.